src/main/java/org/folio/circulation/rules/CirculationRuleMatchEntity.java

```diff
@@ -0,0 +1,22 @@
+package org.folio.circulation.rules;
+
+public class CirculationRuleMatchEntity {
```

Comment:
Similarly, what extra information does the suffix `entity` provide?

Reply:
renamed